Doc/library/os.rst

```diff
@@ -4041,83 +4041,44 @@ written in Python, such as a mail server's external command delivery program.
    number is zero); the high bit of the low byte is set if a core file was
    produced.
 
+   If there are no children that could be waited for, :exc:`ChildProcessError`
+   is raised.
```

Comment:
I'm surprised `ChildProcessError` wasn't already mentioned, but it's definitely good to see it documented now.